pkg/apis/addtoscheme_sources_v1alpha1.go

```diff
@@ -18,9 +18,10 @@ package apis
 
 import (
 	"github.com/knative/eventing-sources/pkg/apis/sources/v1alpha1"
+	servingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
 )
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme)
+	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme, servingv1alpha1.SchemeBuilder.AddToScheme)
```

Comment:
I am not sure this is intended to be used like this.

serving should be included only if the controller requires serving. If we had a pkg/apis/serving/v1alpha1 with an init file like this and you import that package like: `_ "github.com/knative/eventing-sources/pkg/apis/serving/v1alpha1` then it will still do what you want and not force all things that leverage eventing-sources to be tightly coupled to serving as well.

Reply:
The concrete problem I ran into is that the manager didn't have Serving's types registered. While what I did here did fix that issue, it definitely felt wrong and was on my list to revisit. Thanks for the pointer to creating a `pkg/apis` entry for serving!